docs/smart-contracts/install.md

```diff
@@ -0,0 +1,51 @@
+
+- **Install tooling**: Installing additional tooling is optional. However, many toolchain extensions—like the Pact language server plugin (LSP) and the community-contributed `pactup` package manager—provide essential quality-of-life enhancements that improve the developer experience. You should also consider adding access to common blockchain applications—like faucets, wallets, and block explorers—to ensure you have an end-to-end development environment.
+
+- **Set up a local node**: A properly configured blockchain node is a crucial part of your development environment for testing and debugging of projects during development and simulating public deployment in an isolated network environment.
+
+## Before you begin
+
+To prepare for installation, you should verify your development environment meets the following basic requirements
+
+- You have an internet connection and a web browser installed on your local computer.
+- You have an integrated development environment (IDE) or code editor such as [Visual Studio Code](https://code.visualstudio.com/download).
+- You have access to an interactive terminal shell as part of the IDE or code editor you use.
+
+## Pact versions
+
+You can choose to download and install either the latest, stable version of Pact, version 4.x, or the development version of Pact Core, version 5 (beta) for Linux or macOS computers.
+
+- Pact 4.13 is the latest stable release of the Pact smart contract language and interactive interpreter for macOS.
+- Pact Core is a reinvention of the Pact language that focuses on enhanced scalability, maintainability, and performance.
+  Pact Core enables sustainable growth of Pact features within the Kadena ecosystem by offering more modular and maintainable internal structures to better support community development and enhancements to the language and components that rely on it.
+  For more information about Pact Core, see the following resources:
+
+  - [Pact Core repository](https://github.com/kadena-io/pact-5)
+  - [Pact 5 Beta: Improving Developer, Miner, and User Experience](https://www.kadena.io/perspectives/pact-5-beta)
+
+## Installation options
+
+You can download and install the Pact programming language and interactive interpreter locally on your local computer from prebuilt platform-specific binaries or build Pact directly from its source code. 
+You can also run Pact in a browser without installing it as a component in the development network Docker image.
+
+The installation instructions in this section describe how to install Pact locally using prebuilt binaries for each operating system platform.
+
+To build Pact binaries directly from source, download the source code from [Pact Releases](https://github.com/kadena-io/pact/releases), then use Homebrew, Cabal from the Haskell toolchain, or the Nix package manager to build Pact. 
```

Comment:
This is a tricky one. Pact-5 lives in a different repository: https://github.com/kadena-io/pact-5

Also, building from sources, users need a Haskell toolchain that can be brought into scope by using Nix. The instructions on how to do so are currently here: https://github.com/kadena-io/pact-5?tab=readme-ov-file#installing-pact-core

Reply:
To build Pact binaries directly from source, see the instructions in the appropriate repository:

- [Pact Core](https://github.com/kadena-io/pact-5/releases/tag/development-latest)
- [Pact 4.x](https://github.com/kadena-io/pact?tab=readme-ov-file#building-from-source)